test/fixtures/names.yml

```diff
@@ -153,6 +153,7 @@ conocybe_filaris: # 11
   display_name: '**__Conocybe filaris__**'
   search_name: Conocybe filaris
   sort_name: Conocybe filaris
+  synonym:                        # has no synonym
```

Comment:
Affirmatively have no synonyms to ensure durability of test which is added in test/models/name_test.rb